workflows/readsimulator/main.nf

```diff
@@ -72,44 +70,21 @@ def multiqc_report = []
 
 workflow READSIMULATOR {
 
+    take:
+    ch_input
+
+    main:
     ch_versions        = Channel.empty()
-    ch_input           = Channel.fromSamplesheet("input")
     ch_simulated_reads = Channel.empty()
-    ch_taxids          = Channel.empty()
-    ch_accessions      = Channel.empty()
 
     if ( params.fasta ) {
         ch_fasta = Channel.fromPath(params.fasta)
     } else {
-        if ( params.ncbidownload_accessions ) {
-            ch_accessions = Channel.fromPath(params.ncbidownload_accessions)
-        } else if ( params.ncbidownload_taxids ) {
-            ch_taxids = Channel.fromPath(params.ncbidownload_taxids)
-        }
-
-        //
-        // MODULE: Download reference fasta files
-        //
-        NCBIGENOMEDOWNLOAD (
-            [ id:"ncbigenomedownload" ],
-            ch_accessions.ifEmpty([]),
-            ch_taxids.ifEmpty([]),
-            params.ncbidownload_group
-        )
-
-        MERGE_FASTAS (
-            NCBIGENOMEDOWNLOAD.out.fna
-        )
-
-        ch_fasta = MERGE_FASTAS.out.fasta
-            .map {
-                meta, fasta ->
-                return fasta
-            }
+        ch_fasta = Channel.empty()
     }
 
-    if ( params.probe_file ) {
-        ch_probes = Channel.fromPath(params.probe_file)
+    if ( params.probe_fasta ) {
+        ch_probes = Channel.fromPath(params.probe_fasta)
     } else {
```

Comment:
I called it probe_file because it now can be either a fasta file or a bed file. In the target_capture workflow, bedtools is used to get a fasta file from the bed file and reference fasta.